### PR TITLE
Use faker everywhere

### PR DESCRIPTION
**Pull request by XVincentX**
We already have a random data generator library, so we can get rid of chance

### CODE REVIEW

File: packages/http/src/mocker/negotiator/__tests__/NegotiatorHelpers.spec.ts [hunk 1]
```diff
@@ -850,7 +847,7 @@ describe('negotiateByPartialOptionsAndHttpContent()', () => {
         dynamic: true,
       };
       const httpContent: IMediaTypeContent = {
-        mediaType: `${chance.string(chanceOptions)}/${chance.string(chanceOptions)}`,
+        mediaType: faker.system.mimeType(),
```
Comment (pytlesk4) [inline packages/http/src/mocker/negotiator/__tests__/NegotiatorHelpers.spec.ts:850]: Nice

File: packages/http-server/src/server.ts [hunk 2]
```diff
@@ -90,7 +91,7 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
     pipe(
       TE.fromEither(requestConfig),
       TE.chain(requestConfig => prism.request(input, operations, requestConfig)),
-      TE.chain(response => {
+      TE.chainIOEitherK(response => {
```
Comment (pytlesk4) [inline packages/http-server/src/server.ts:94]: What's the difference between `chain` and `chainIOEitherK`?

**Review by pytlesk4 — APPROVED** [review]
Just a small question, but love the use of faker.

File: packages/http-server/src/server.ts (hunk 2 above)
Comment (XVincentX) [reply packages/http-server/src/server.ts:94]: TE.chain requires the function to return a `TaskEither`; in this part we're internally returning an `Either` instead and so a transformation was required: `return TE.fromIOEither(r)`

fp-ts 2.4 introduced the ability of returning an Either from a TaskEither chain using the dedicated function, so we are not handing the transformation anymore.

**Pull request merged**